code/client/cl_main.cpp

```diff
@@ -1258,6 +1259,7 @@ void CL_Init( void ) {
 	cl_run = Cvar_Get ("cl_run", "1", CVAR_ARCHIVE_ND);
 	cl_sensitivity = Cvar_Get ("sensitivity", "5", CVAR_ARCHIVE);
 	cl_mouseAccel = Cvar_Get ("cl_mouseAccel", "0", CVAR_ARCHIVE_ND);
+	cl_mouseAspectScaling = Cvar_Get ("cl_mouseAspectScaling", "0", CVAR_ARCHIVE_ND);
```

Comment:
Since this is UI only, I'd actually prefer this to be on by default.

Reply:
I would prefer to keep it off by default. At least from a modding perspective it's probably not nice if you switch between different clients for testing and the mouse acts different in one of them without a menu option to change the behavior.

Reply:
Hm, yea. Valid point I agree with.